components/script/dom/webglrenderingcontext.rs

```diff
@@ -1039,6 +1039,147 @@ impl WebGLRenderingContextMethods for WebGLRenderingContext {
     }
 
     // https://www.khronos.org/registry/webgl/specs/latest/1.0/#5.14.10
+    fn Uniform2fv(&self,
+                  _cx: *mut JSContext,
+                  uniform: Option<&WebGLUniformLocation>,
+                  data: Option<*mut JSObject>) {
+        let data = match data {
```

Comment:
I suppose so, yes. The other previously implemented uniform vector functions didn't do it either, so should I fix that in this PR, or open an issue and handle it in another PR?